addons/l10n_ke/models/l10n_ke_item_code.py

```diff
@@ -0,0 +1,16 @@
+
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models, fields
+
+
+class L10nKeItemCode(models.Model):
+    _name = 'l10n_ke.item.code'
+    _rec_names_search = ['code', 'description']
+
+    code = fields.Char(string='KRA Item Code')
```

Comment:
maybe _rec_name='code'?

Reply:
do you mean `_rec_name = 'code'` rather than defining the `name_get`... because with the `name_get` defined, the `_rec_name` does nothing.

Reply:
So https://github.com/odoo/odoo/blob/9afe7c74c9c59b2a0fa4b1669dcc85da0d2225ee/addons/mrp/models/mrp_bom.py#L18 is wrong?